paasta_tools/mesos_tools.py

```diff
@@ -615,3 +615,36 @@ def get_mesos_network_for_net(net):
         'host': 'HOST',
     }
     return docker_mesos_net_mapping.get(net, net)
+
+
+def get_mesos_task_count_by_slave(mesos_state, pool='default'):
+    """Get counts of running tasks per mesos slave. Also include separate count of chronos tasks
```

Comment:
So my thinking was that batches are more difficult to "drain". In the sense that the best you can do is wait for the current run to finish. So I'm trying to sort by `chronos_count` and then the total number of tasks.